src/info.plist

```diff
@@ -63,16 +42,16 @@
 				<key>script</key>
 				<string>curl "https://api.todoist.com/rest/v1/tasks" \
     -X POST \
-    --data '{"content": "{query}", "due_lang": "en"}' \
+    --data '{"content": "${1}", "due_lang": "en"}' \
```

Comment:
```suggestion
    --data '{"content": "'$1'", "due_lang": "en"}' \
```

It looks like that there is need to add `'` around `$1`, otherwise the param from argv will not be entropolated.

Reply:
Good catch. On second look, because you’re sending a string in this case it will need escaping, though the one you had would escape unintended things, such as `$`. It should escape what matters to JSON, double quotes and backslashes. Updated accordingly.